### PR TITLE
* Upgrade to the new LPI structure keeping only necessary info on it.

### PR DESCRIPTION
**Pull request by silvioprog**

### CODE REVIEW

File: source/component/pasdoc.css.inc [hunk 1]
```diff
@@ -131,6 +131,7 @@
 '     is closely related to content below. In this situation' + LineEnding +
 '     (where the font size is just as a normal text), smaller bottom' + LineEnding +
 '     margin seems to look good. */' + LineEnding +
+'  margin-top: 1.4em;' + LineEnding +
```
Comment (michaliskambi) [inline source/component/pasdoc.css.inc]: Note: This should be edited in the pasdoc.css (without the ".inc") file also.

The "pasdoc.css.inc" file is automatically generated from "pasdoc.css" by running "make" in the source/component/ directory. It uses the file_to_pascal_string binary that can be compiled by doing "make build-tools" in the top-level pasdoc directory (it's trivial source code is in source/tools/ ).

Comment (silvioprog) [inline source/component/pasdoc.css.inc]: Oh, you are right, I'm going to do it and open a pull request ...

Comment (silvioprog) [inline source/component/pasdoc.css.inc]: Done. :-)

**Comment by michaliskambi** [comment]
All cool, merged!

**Pull request merged**